readthedocs/vcs_support/backends/git.py

```diff
@@ -66,21 +66,21 @@ def checkout_revision(self, revision=None):
             revision = 'origin/%s' % branch
 
         code, out, err = self.run('git', 'checkout',
-                                  '--force', '--quiet', revision)
+                           '--force', revision)
```

Comment:
Since we are exposing the output to the user, I removed all the `quiet` arguments.